src/Avalonia.Base/Input/GestureRecognizers/GestureRecognizer.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Avalonia.Input.GestureRecognizers
+{
+    public abstract class GestureRecognizer : StyledElement
+    {
+        public abstract IInputElement? Target { get; }
+        public abstract void Initialize(IInputElement target);
```

Comment:
Might be best to make these methods `protected internal`? Use code shouldn't be calling them I don't think?